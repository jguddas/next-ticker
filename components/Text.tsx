import React from 'react'
import { x } from '@xstyled/styled-components'

interface Props extends React.ComponentProps<typeof x.div> {
  value?: string
}

export default ({ children, ...props }: Props): JSX.Element => (
  <x.div color="text" fontSize lineHeight {...props}>
    {children}
  </x.div>
)
