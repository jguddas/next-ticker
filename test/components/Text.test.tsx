import React from 'react'
import { render } from '../testUtils'
import Text from '../../components/Text'
import 'jest-styled-components'

describe('Text', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Text>Hello World</Text>, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
