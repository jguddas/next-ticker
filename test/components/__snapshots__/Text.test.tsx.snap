// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text matches snapshot 1`] = `
<DocumentFragment>
  .c0.c0 {
  color: text;
  font-size: 1px;
}

<div
    class="c0"
  >
    Hello World
  </div>
</DocumentFragment>
`;
